### ISSUE TITLE
PHP8.1 (ZTS+FPM+PARALLEL)

### ISSUE THREAD

**Issue by AStroDevOPS** [krakjoe/parallel#246]
I've got some problem with parallel when compiling php 8.1.11
```
/tmp/pear/temp/parallel/src/runtime.c: In function 'zm_startup_PARALLEL_RUNTIME':
/tmp/pear/temp/parallel/src/runtime.c:161:42: error: 'zend_class_serialize_deny' undeclared (first use in this function); did you mean 'zend_ce_serializable'?
  161 |     php_parallel_runtime_ce->serialize = zend_class_serialize_deny;
      |                                          ^~~~~~~~~~~~~~~~~~~~~~~~~
      |                                          zend_ce_serializable
/tmp/pear/temp/parallel/src/runtime.c:161:42: note: each undeclared identifier is reported only once for each function it appears in
/tmp/pear/temp/parallel/src/runtime.c:162:44: error: 'zend_class_unserialize_deny' undeclared (first use in this function); did you mean 'zend_unserialize_data'?
  162 |     php_parallel_runtime_ce->unserialize = zend_class_unserialize_deny;
      |                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                            zend_unserialize_data
make: *** [Makefile:219: src/runtime.lo] Error 1
ERROR: `make' failed
```

**Comment by cmb69** [comment]
Duplicate of #215. There is #245 which is supposed to fix that, but there are some unresolved issues.
